<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="selectionLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1440</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid black; border-radius:5%;background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0)); color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="dfsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{color:black;} QRadioButton::indicator{background-color:white; border-radius:5;;} QRadioButton::indicator::checked{ background-color:lime;} QRadioButton::indicator::unchecked{background-color:black;}</string>
   </property>
   <property name="text">
    <string>Depth-First-Search</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bfsButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{color:black;} QRadioButton::indicator{background-color:white; border-radius:5;;} QRadioButton::indicator::checked{ background-color:lime;} QRadioButton::indicator::unchecked{background-color:black;}</string>
   </property>
   <property name="text">
    <string>Breadth-First-Search</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="astarButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{color:black;} QRadioButton::indicator{background-color:white; border-radius:5;;} QRadioButton::indicator::checked{ background-color:lime;} QRadioButton::indicator::unchecked{background-color:black;}</string>
   </property>
   <property name="text">
    <string>A-Star</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="dijkstraButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{color:black;} QRadioButton::indicator{background-color:white; border-radius:5;;} QRadioButton::indicator::checked{ background-color:lime;} QRadioButton::indicator::unchecked{background-color:black;}</string>
   </property>
   <property name="text">
    <string>Dijkstra</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>30</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69)); color:black; border-radius:15%;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wallsButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69)); color:black; border-radius:15%;</string>
   </property>
   <property name="text">
    <string>Clear Walls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pathButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>15</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69)); color:black; border-radius:15%;</string>
   </property>
   <property name="text">
    <string>Clear Path</string>
   </property>
  </widget>
  <widget class="QSlider" name="animationSpeedSlider">
   <property name="geometry">
    <rect>
     <x>1290</x>
     <y>40</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {  border: 1px solid #999999;
    height: 8px; background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #deb887, stop:1 #daa520);  margin: 2px 0;
    border-radius:5%;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #bdb76b, stop:1 #f0e68c);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0;  border-radius: 3px;
}</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
   <property name="sliderPosition">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>20</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white</string>
   </property>
   <property name="text">
    <string>animation Speed</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="bestfsButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{color:black;} QRadioButton::indicator{background-color:white; border-radius:5;;} QRadioButton::indicator::checked{ background-color:lime;} QRadioButton::indicator::unchecked{background-color:black;}</string>
   </property>
   <property name="text">
    <string>Best-First-Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forestButton">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>50</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69)); color:black; border-radius:15%;</string>
   </property>
   <property name="text">
    <string>Clear Forest</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="wallsCheckbox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{ color:black; }QCheckBox::indicator:unchecked {background-color:black;}  QCheckBox::indicator:checked { background-color:lime;}</string>
   </property>
   <property name="text">
    <string>Walls</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="forestCheckbox">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>50</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox{ color:black; }QCheckBox::indicator:unchecked {background-color:black;}  QCheckBox::indicator:checked { background-color:lime;}</string>
   </property>
   <property name="text">
    <string>Forest</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mazeButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>15</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69)); color:black; border-radius:15%;</string>
   </property>
   <property name="text">
    <string>Maze</string>
   </property>
  </widget>
  <widget class="QSlider" name="mazeSlider">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>15</y>
     <width>20</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:vertical {  border: 1px solid #999999;  width: 12px; background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #deb887, stop:1 #daa520);  margin: 2px 0;
    border-radius:5%;
}

QSlider::handle:vertical {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #bdb76b, stop:1 #f0e68c);
    border: 1px solid #5c5c5c;
    height: 10px;
    margin:0 ;  border-radius: 3px;
}</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>7</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="mazeIncrement">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>15</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mazeDecrement">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>65</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="roomsLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>25</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; background:black; border-radius:8%;</string>
   </property>
   <property name="text">
    <string>rooms&lt;br&gt;4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
